### PR TITLE
New working member announcement

### PR DESCRIPTION
**Pull request by tellison**

### CODE REVIEW

**Review by smlambert — APPROVED** [review]
LGTM

File: content/blog/adoptium-welcomes-google/index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+title: Adoptium Welcomes Google
+date: "2022-10-14T12:00:00+00:00"
+author: pmc
+description: Adoptium welcomes Google as the latest working group member.
+tags:
+  - "Working Group"
+  - "Eclipse Temurin"
+---
+
+### Adoptium welcomes Google Cloud
```
Comment (karianna) [inline content/blog/adoptium-welcomes-google/index.md:11]: Is it Google Cloud or Google as the legal entity?

**Review by karianna — APPROVED** [review]
LGTM

**Review by jerboaa — APPROVED** [review]

**Pull request merged**